### PR TITLE
fix: downloads and services affected by android 15 support

### PR DESCRIPTION
**Pull request by IndusAryan**

### CODE REVIEW

**Comment by Luna712** [comment]
It still crashes with this patch.

File: app/src/main/java/com/lagradost/cloudstream3/services/VideoDownloadService.kt [hunk 1]
```diff
@@ -34,13 +44,19 @@ class VideoDownloadService : Service() {
             }
         }
 
+        VideoDownloadManager.startServiceForeground(this)
```
Comment (fire-light42) [inline app/src/main/java/com/lagradost/cloudstream3/services/VideoDownloadService.kt:47]: The video download manager is already running using the worker foreground service:
```
Started foreground service Intent { act=ACTION_START_FOREGROUND cmp=com.lagradost.cloudstream3.prerelease.debug/androidx.work.impl.foreground.SystemForegroundService (has extras) }
```
Starting another service (without proper termination) is unnecessary

Comment (fire-light42) [reply app/src/main/java/com/lagradost/cloudstream3/services/VideoDownloadService.kt:47]: I will remove personally due to the urgency of a fix

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -207,18 +207,25 @@
...
+            android:foregroundServiceType="dataSync"
+            tools:node="merge" />
+
         <activity
             android:name=".ui.ControllerActivity"
             android:exported="false" />
 
         <service
+            android:name=".services.PackageInstallerService"
+            android:enabled="false"
```
Comment (fire-light42) [inline app/src/main/AndroidManifest.xml:227]: Why disabled?

**Comment by fire-light42** [comment]
Let us hope it works this time

**Comment by Luna712** [comment]
@fire-light42 seems PackageInstaller is still broke:
```
Fatal exception on thread main (2) java.lang.RuntimeException: Unable to stop service com.lagradost.cloudstream3 .services.PackageInstallerService@af1a8d7: java.lang.IllegalArgumentException:

Receiver not registered: com.lagradost cloudstream3.utils. Apkinstaller$installActio nReceiver$1@8356a64
```

**Pull request merged**